buildSrc/src/main/kotlin/buildsrc/utils/javaToolchains.kt

```diff
@@ -11,6 +11,10 @@ import org.jetbrains.kotlin.gradle.dsl.JvmTarget
 
 // Utilities for configuring Java Toolchains
 
+object JavaVersions {
+    const val JAVA_TOOLCHAIN_VERSION = 21
```

Comment:
A way to share this constant between both usages. Let me know if there's a better approach.

Reply:
@OptimumCode any suggestions here? It's itching me a bit, maybe version catalog is the best approach (not sure how easy to access from all places)? Ideally I'd like us to keep all Java versions in a single place, so: toolchain version (21), target release version (8), target version used in tests (11).

Reply:
Yes, I also thought about the version catalog. There is a guide on how to use the same version catalog in both main project and buildSrc: https://docs.gradle.org/current/userguide/version_catalogs.html#sec:buildsrc-version-catalog (not the best syntax but more or less fine, I think)